metadata-ingestion-modules/airflow-plugin/tests/integration/test_plugin.py

```diff
@@ -373,20 +377,22 @@ def test_airflow_plugin(
         print("Sleeping for a few seconds to let the plugin finish...")
         time.sleep(10)
 
-    _sanitize_output_file(airflow_instance.metadata_file)
-
-    check_golden_file(
-        pytestconfig=pytestconfig,
-        output_path=airflow_instance.metadata_file,
-        golden_path=golden_path,
-        ignore_paths=[
-            # TODO: If we switched to Git urls, maybe we could get this to work consistently.
-            r"root\[\d+\]\['aspect'\]\['json'\]\['customProperties'\]\['datahub_sql_parser_error'\]",
-            r"root\[\d+\]\['aspect'\]\['json'\]\['customProperties'\]\['openlineage_.*'\]",
-            r"root\[\d+\]\['aspect'\]\['json'\]\['customProperties'\]\['log_url'\]",
-            r"root\[\d+\]\['aspect'\]\['json'\]\['externalUrl'\]",
-        ],
-    )
+    # Golden file will NOT get generated for the `filtered DAG`
+    if dag_id != NAME_OF_DAG_TO_FILTER_FROM_INGESTION:
```

Comment:
we need to check that the golden file is missing / empty when the dag_id is NAME_OF_DAG_TO_FILTER_FROM_INGESTION? otherwise, this test doesn't actually do anything